src/main/java/hu/wurfel/refference/school/diary/ResponseCreationClarificationDiary.java

```diff
@@ -10,135 +10,135 @@
 @Service
 public class ResponseCreationClarificationDiary extends ResponseCreationClarificationTemplate {
 
-    public ArrayList create(SearchRequest request) {
+    public ArrayList create(DiaryRequest diaryRequest) {
         rContent = new ArrayList<>();
-        switch (request.getRWith()) {
+        switch (diaryRequest.getSearchWith()) {
```

Comment:
Dear Mother of God... erről személyesen beszéljünk. :D

Reply:
Bár ez az osztály már nincs jelen, maga ez a switch, jelentős változtatásokkal, de megtalálható a controlerben, segéd metódusként

Reply:
Látom vereted rendesen. 😄 
Kommenteltem oda is.